# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/WebCodecsUtils.h
# Commit: bcb45500ba22
# Full Hash: bcb45500ba2223a5e21bcc29922c1260d350694f
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 3
# Description:
#   Bug 1749047 - Wire up more codecs from VideoEncoder to the new FFmpeg PEM. r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D196108
# ==============================================================================

diff -r a78bb8799ef4 -r bcb45500ba22 dom/media/webcodecs/WebCodecsUtils.h
--- a/dom/media/webcodecs/WebCodecsUtils.h	Fri Dec 22 21:39:12 2023 +0000
+++ b/dom/media/webcodecs/WebCodecsUtils.h	Fri Dec 22 21:39:13 2023 +0000
@@ -18,6 +18,7 @@
 #include "mozilla/dom/UnionTypes.h"
 #include "mozilla/dom/VideoEncoderBinding.h"
 #include "mozilla/dom/VideoFrameBinding.h"
+#include "PlatformEncoderModule.h"
 
 namespace mozilla {
 
@@ -224,6 +225,8 @@
 VideoColorSpaceInit FallbackColorSpaceForVideoContent();
 VideoColorSpaceInit FallbackColorSpaceForWebContent();
 
+Maybe<CodecType> CodecStringToCodecType(const nsAString& aCodecString);
+
 } // namespace dom
 
 }  // namespace mozilla
